beets/library.py

```diff
@@ -186,6 +186,7 @@ def parse(self, key):
         for flat, sharp in self.ENHARMONIC.items():
             key = re.sub(flat, sharp, key)
         key = re.sub(r'[\W\s]+minor', 'm', key)
+        key = re.sub(r'[\W\s]+major', '', key)
```

Comment:
Seems like a good policy. We should write tests for this (not necessarily now)—it would be good to show that many "natural" ways to write musical keys get correctly canonicalized.